.editorconfig

```diff
@@ -1,7 +1,8 @@
 root = true
 
 [*]
-indent_style = tab
+indent_style = space
+indent_size = 3
```

Comment:
My concern with changing this is going forward every PR we make will be changing the formatting of every single file.